modules/microshift-observe-debug-etcd-server.adoc

```diff
@@ -0,0 +1,29 @@
+// Module included in the following assemblies:
+//
+//* microshift_support/microshift-etcd.adoc
+
+:_content-type: PROCEDURE
+[id="microshift-observe-debug-etcd-server_{context}"]
+= Observe and debug the {product-title} etcd server
+
+You can gather `journalctl` logs to observe and debug the etcd server logs.
+
+.Prerequisites
+
+* You have installed the OpenShift CLI (oc).
+
+* You have run the command `systemctl start microshift` as part of the {product-title} start-up.
```

Comment:
I would say that we need to have MicroShift service running as a prerequisite without mentioning the command. There is more than one way to start the service, so mentioning something concrete here is a bit misleading.